microcosm_flask/routing.py

```diff
@@ -57,16 +57,14 @@ def route(path, operation, ns):
         """
         def decorator(func):
             endpoint = ns.endpoint_for(operation)
+            endpoint_path = make_path(graph, path)
```

Comment:
Unrelated: I've found myself using this value for debugging here or there, so I figured it was better to calculate it earlier.